cmake/patches/zookeeper.3.4.14.buf

```diff
@@ -0,0 +1,3767 @@
+/**
```

Comment:
This is a patch file. I should shorten this to use a diff'd patch

Reply:
What is this patching?